t/resolver.t

```diff
@@ -34,6 +34,15 @@ GET /t
 nameservers: 3 127.0.1.15353 1.2.3.453 4.5.6.753
 --- user_files
 >>> resolv.conf
+# nameserver updated  in comentary
```

Comment:
This is good. It's testing more things than before, so I think it's worth expanding the description of the test (line 16).

Reply:
done in d49b712